server/portal/apps/workspace/api/utils.py

```diff
@@ -36,14 +36,15 @@ def check_job_for_timeout(job):
     since Tapis does not have native support for interactive jobs yet
     """
 
-    if hasattr(job, "notes"):
-        notes = _get_job_notes(job.notes)
```

Comment:
`_get_job_notes` is only used here. If no longer needed, let's delete that method, or refactor the method to include this updated logic.